content/en/docs/reference/labels-annotations-taints.md

```diff
@@ -181,6 +181,16 @@ Used on: PersistentVolumeClaim
 
 This annotation will be added to dynamic provisioning required PVC.
 
+## volume.beta.kubernetes.io/mount-options (deprecated) {#mount-options}
+
+Example : `volume.beta.kubernetes.io/mount-options: ["ro", "soft"]`
+
+Used on: PersistentVolume
+
+A Kubernetes administrator can specify additional [mount options](/docs/concepts/storage/persistent-volumes/#mount-options) for when a Persistent Volume is mounted on a node.
+
+This annotation has been deprecated.
```

Comment:
Is the annotation promoted into a field or is it merely abandoned?

Reply:
@kubernetes/sig-storage-misc - who could answer this?